examples/open-ai-chat-plugin/packages/service/src/todo.ts

```diff
@@ -0,0 +1,96 @@
+import { extendApi } from "@anatine/zod-openapi";
+import { command, entity } from "@eventual/core";
+import { ulid } from "ulidx";
+import z from "zod";
+
+const userNameSchema = extendApi(z.string(), {
+  description: "The name of the user",
+});
+
+const todoIdSchema = extendApi(z.string(), {
+  description: "Id of a single todo item",
+});
```

Comment:
This is so ugly. Why can't we use the built in zod description?

```ts
z.string().description("ID of a single TODO item")
```

Reply:
It doesn't work unfortunately.